app/views/shared/_skiplinks.html.haml

```diff
@@ -2,8 +2,8 @@
   %nav.fr-container{ 'aria-label': t('.fast_access'), role: 'navigation' }
     %ul.fr-skiplinks__list
       %li
-        = link_to t('.content'), '#content', class: 'fr-link'
+        = link_to t('.content'), '#content', class: 'fr-link', data: { turbolinks: false }
```

Comment:
j'ai toujours un trou sur ce que ça fait, le côté "turbolinks or not turbolinks". Je sais un moment, et puis qq mois + tard j'oublie. Ca me fatigue.

Reply:
ça essaie d'accélérer le chargement des pages au clic sur des liens avec du cache et je ne sais plus trop quoi. Pour faire un site plus rapide à charger. Mais ici ça faisait des trucs bizarre, la page se rechargeait et n'allait pas à l'ancre.